stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh

```diff
@@ -6,7 +6,7 @@ set -e
 DOCKERFILE_DIR=$( cd $(dirname "${BASH_SOURCE[0]}") >/dev/null 2>&1 && pwd )
 STRATUM_ROOT=${STRATUM_ROOT:-"$( cd "$DOCKERFILE_DIR/../../../../.." >/dev/null 2>&1 && pwd )"}
 STRATUM_BF_DIR=$( cd "$DOCKERFILE_DIR/.." >/dev/null 2>&1 && pwd )
-STRATUM_TARGET=${STRATUM_TARGET:-stratum_bf}
+STRATUM_TARGET=${STRATUM_TARGET:-stratum_bfrt}
```

Comment:
stratum_bf is still the default
```suggestion
STRATUM_TARGET=${STRATUM_TARGET:-stratum_bf}
```